components/styled/table/src/molecules/table-cell.tsx

```diff
@@ -28,6 +28,9 @@ const TableCell: FC<CellProps> = ({
   }
 
   if (adminActions) {
+    if (adminActions.length === 0) {
+      return <></>
```

Comment:
Very nitpicky but technically returning null is preferable to the empty fragment